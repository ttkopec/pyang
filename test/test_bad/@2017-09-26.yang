module noname {
    namespace "urn:noname";
    prefix "noname";
    revision 2017-09-26 {
    }
}
