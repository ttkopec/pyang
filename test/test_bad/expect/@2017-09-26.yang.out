@2017-09-26.yang:1: warning: FILENAME_BAD_MODULE_NAME
